src/PlaneWaveBasis.jl

```diff
@@ -67,10 +67,15 @@ struct PlaneWaveBasis{T <: Real}
     fft_size::Tuple{Int, Int, Int}
 
     # Plans for forward and backward FFT
+    # These plans follow DFTK conventions (see above)
     opFFT  # out-of-place FFT plan
     ipFFT  # in-place FFT plan
     opIFFT
     ipIFFT
+    opFFT_unnormalized   # unnormalized plans
+    ipFFT_unnormalized
+    opBFFT_unnormalized  # "backward" FFT in FFTW terminology: no normalization at all
+    ipBFFT_unnormalized
```

Comment:
This starts to become a zoo. Why don't we do the normalisation in a post-step as a `./= norm` ?

Reply:
Alternatively some sort of struct / named tuple(s) to hold these guys would be helpful.

Reply:
I don't understand what you mean. This is directly used in the G_to_r! functions, so should be precomputed.

Reply:
We could, and have a function to construct it. Can't be bothered for now though.

Reply:
No I meant after the G_to_r! you use the normalisation as a post-step if normalisation is needed. So we no longer have any "scaled plans". That was my suggestion.

Reply:
Hm, I see. I think it's nicer to have these normalized plans following DFTK conventions and the unnormalized ones as an optimization

Reply:
Ok then at least split them up into two blocks of four to make it clear that the "these follow DFTK's convention" only apply to the first four blocks.